aplib.net-demo/Assets/Scripts/Entities/EntitySound.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine;
+
+[RequireComponent(typeof(AudioSource))]
+public class EntitySound : MonoBehaviour
+{
+    [SerializeField] private AudioClip[] _footSteps;
+    [SerializeField] private AudioClip[] _knifeSwings;
+    [SerializeField] private AudioClip[] _crossbowShots;
+    [SerializeField] private AudioClip[] _deathSounds;
```

Comment:
```suggestion
    [SerializeField]
    private AudioClip[] _footSteps;
    
    [SerializeField]
    private AudioClip[] _knifeSwings;
    
    [SerializeField]
    private AudioClip[] _crossbowShots;
    
    [SerializeField]
    private AudioClip[] _deathSounds;
```

Reply:
There is a resharper setting for this in the editorconfig, consider using resharper.